app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  has_many :events, foreign_key: :creator_id, class_name: 'Event'
+  has_many :attendance, foreign_key: :attendee_id
```

Comment:
I believe the attendance should be plural because of has_many please make sure of that